``_pytest._io`` ImportError

I ran into an exception while running the tests:

```
(37B) [corbett8@lassen709:psi-j-python]$ make tests -- --upload-results
PYTHONPATH=/usr/WS2/corbett8/ExaWorks/psi-j-python/src: python -m pytest -v --upload-results
ImportError while loading conftest '/usr/WS2/corbett8/ExaWorks/psi-j-python/tests/conftest.py'.
tests/conftest.py:20: in <module>
    from _pytest._io import TerminalWriter
E   ImportError: cannot import name 'TerminalWriter' from '_pytest._io' (/usr/tce/packages/python/python-3.7.2/lib/python3.7/site-packages/_pytest/_io/__init__.py)
make: *** [Makefile:14: tests] Error 4
```

The problem went away when I upgraded ``pytest`` from 4.3 to 6.2. So the easiest solution would be to require ``pytest >= X`` or similar in our requirements file. But I was wondering if there are other options available to us that wouldn't require ``_pytest`` imports. We could replace the line with ``from py.io import TerminalWriter`` but ``py`` is in maintenance mode. Any thoughts?

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -7,7 +7,7 @@
 
 # Testing / QA
 mypy >=0.790
-pytest
+pytest >= 6.2.0
 flake8
 autopep8
 types-requests

